doc/howto/network_forwards.md

```diff
@@ -50,7 +51,7 @@ Network forwards have the following properties:
 
 Property         | Type       | Required | Description
 :--              | :--        | :--      | :--
-`listen_address` | string     | yes      | IP address to listen on
+`listen_address` | string     | no       | IP address to listen on
```

Comment:
Just marking this because this change will collide with https://github.com/canonical/lxd/pull/13355